### ISSUE TITLE
Memory of 32 gigabytes (GB) is misleading as it doesn't take into account user volumes or modalities being used

### ISSUE THREAD

**Issue by CraigBushy** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#469]
This document states Front End Servers, Back End Servers, Standard Edition servers, and Persistent Chat Servers all must have 32GB RAM regardless of the user volume and modalities being consumed. This contradicts the Skype Calculator here: https://www.microsoft.com/en-us/download/details.aspx?id=51196  which calculates the exact requirement. This is misleading and results in over spec'd servers and cost for no reason.  Please clarify in what scenario 32GB is the recommend amount of RAM.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7c475b5d-09ed-7f58-86d4-27f28e33ef7a
* Version Independent ID: f0ad50f3-ea93-9424-6de8-a0b778e90b23
* Content: [Server requirements for Skype for Business Server 2015](https://docs.microsoft.com/en-us/skypeforbusiness/plan-your-deployment/requirements-for-your-environment/server-requirements)
* Content Source: [Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md)
* Product: **skype-for-business-itpro**
* GitHub Login: @MicrosoftHeidi
* Microsoft Alias: **heidip**

**Comment by kenwith** [comment]
@CraigBushy - this is a very good point you bring up and we will have to figure this out from product team regarding the recommendations and get back to you. Thank you for taking the time to submit this and sorry for the long delay.

**Comment by kenwith** [comment]
@CraigBushy - sorry for the long delay on this. The Skype writing team is currently working on the 2019 release and I was able to get this into the backlog for the team and they will address with the 2019 content and also the 2015 as you pointed. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
